### DISCUSSION TITLE
剛拿到新電腦要怎麼測試? 3DMark來告訴你!

### DISCUSSION THREAD

**Discussion by a45320614** [1101-GS4719-C/Python-Community-GS4719-C#697] [General]
今天我要來分享的是叫做3DMark的效能測試軟體

![image](https://user-images.githubusercontent.com/92359154/191442161-519c5b14-8a41-4bb3-9e9f-e6242dad0401.png)

3DMark是一款Steam上的效能測試軟體
可以幫助測試裝置的3D圖形渲染效能或是CPU核心處理能力等等
為什麼3DMark可以用來測試新電腦呢?

因為3DMark會進行一種俗稱「燒機」的測試
在測試期間，3DMark會用不同的場景讓你的電腦以高效能運作，以此來測試電腦的效能極限!
由於測試過程中電腦會發燙，也因此出現了「燒機」這樣的稱呼
測試完後，會獲得各項測驗的分數，以此來檢視電腦的效能是不是符合內部的零組件應有的能力
如果分數差不多，不論你是想玩3A大作，或是剪輯4K影片都可以通通放心啦
除了電腦之外，3DMark也有在iOS及Android上推出哦~

Steam頁面連結:
https://store.steampowered.com/app/223850/3DMark/?l=tchinese

**Comment by Yunhans** [comment]
我還沒開3DＭark電腦就燒起來了

**Reply by a45320614** [reply]
那你應該是太夯了吧哥

**Comment by danwan1027** [comment]
電腦不小心燒掉了可以獲得賠償嗎？

**Comment by williamyen0623** [comment]
窮窮的學生 破破的電腦跑3DMARK電腦直接燒掉。＿。

**Comment by Chiway1229** [comment]
我試了3Dmark 電腦直接燒爛，我可以拿理賠嗎

**Comment by evanchou1204** [comment]
文書機完全不需要考慮燒機的問題，正常玩遊戲就快燒掉了
